results.tex

```diff
@@ -11,20 +11,21 @@ \subsection{Nuclear Material Inventory}
 \begin{table}[h]
 	\centering
 %	\scalebox{0.86}{
-		\begin{tabular}{ccc}
+	\caption{Nuclear material inventory in the \gls{EU} in 2050 is
+                  summarized. The \gls{UOX} in the simulation is either stored or reprocessed to create \gls{MOX}.}
+
+		\begin{tabular}{lrl}
 			\hline
-			\textbf{Category } & \textbf{Value [MTHM]} & \textbf{Specifics}\\ \hline
-			UOX Usage  & 158,794 &  \\ 
-			MOX Usage  & 6,671  & \\ 
-			\textbf{Used UOX Stored}  & \textbf{95,161}  & \gls{UNF} that is not reprocessed\\
-			\textbf{Used UOX Stored (France)} & \textbf{9,979}  & \gls{UNF} that is not reprocessed \\
-			Tails  & 979,463  & \\ 
+			\textbf{Category} & \textbf{Value [MTHM]} & Specifics \\ \hline
+			UOX Loaded  & 158,794 & UOX that is loaded into the reactors until 2050 in all EU nations \\ 
+			MOX Loaded  & 6,671  & MOX that is loaded in to the reactors until 2050 in all French reactors \\ 
+			\textbf{Used UOX Available for Reprocessing (EU except France)}  & \textbf{95,161}  & Used UOX from all EU nations except for France that is stored to be reprocessed for ASTRID MOX production.\\
```

Comment:
maybe add parenthesis around "except for france". It's a bit of a confusing run-on as is.